editor/state.js

```diff
@@ -89,59 +89,72 @@ export const editor = combineUndoableReducers( {
 					};
 				}, omit( state, action.uids ) );
 
-			case 'REMOVE_BLOCK':
-				return omit( state, action.uid );
+			case 'REMOVE_BLOCKS':
+				return omit( state, action.uids );
 		}
 
 		return state;
 	},
 
 	blockOrder( state = [], action ) {
-		let index;
-		let swappedUid;
 		switch ( action.type ) {
 			case 'RESET_BLOCKS':
 				return action.blocks.map( ( { uid } ) => uid );
 
-			case 'INSERT_BLOCK':
+			case 'INSERT_BLOCK': {
```

Comment:
Curious to know why do you wrap `case` between `{}`. Is this a good practice I'm missing?